### PR TITLE
Improve playlist plugin docs

### PR DESCRIPTION
**Pull request by JOJ0**
## Description

Improve documentation around the playlist plugin. Clarify what it does and what it does not.

### CODE REVIEW

File: docs/plugins/playlist.rst [hunk 1]
```diff
@@ -24,6 +24,16 @@ name::
 
     $ beet ls playlist:anotherplaylist
 
+A playlist query will use the paths found in the playlist file to match items
+in the beets library. Since `playlist:` submits a regular beets query like any
```
Comment (sampsyo) [inline docs/plugins/playlist.rst:28]: Maybe consider linking to the [query page](https://github.com/beetbox/beets/blob/master/docs/reference/query.rst)? Then we probably don't need the "e.g." parenthetical to explain more.

File: docs/plugins/playlist.rst [hunk 2]
```diff
@@ -24,6 +24,16 @@ name::
...
+other query syntax (eg. `artist:`, `year:`, ...), the sorting order of either
+the configuartion setting `sort_item` or any sorting patterns submitted with
+the query command will be used::
```
Comment (sampsyo) [inline docs/plugins/playlist.rst:31]: Maybe this should be phrased as: "If you want the list in any particular order, you can use the standard beets query syntax for sorting:"

File: docs/plugins/playlist.rst [hunk 3]
```diff
@@ -24,6 +24,16 @@ name::
...
+
+    $ beet ls playlist:/path/to/someplaylist.m3u artist+ year+
+
+Currently retaining the original sort order of the playlist is not supported.
```
Comment (sampsyo) [inline docs/plugins/playlist.rst:35]: To avoid passive voice: "Playlist queries do not reflect the original order in the `m3u` file."

**Review by sampsyo — COMMENTED** [review]
Thank you!! Here are some ideas for wording for maximizing clarity.

**Comment by JOJ0** [comment]
Thanks a lot for your suggestions. Tried to incorporate them. That's the new version rendered:
image

**Comment by sampsyo** [comment]
Looks excellent! Thanks for iterating!!

**Pull request merged**